### ISSUE TITLE
Enable free text search

### ISSUE THREAD

**Issue by haayman** [elk-zone/elk#2410]
Right now only searching for hashtags and users is enabled, but now mastodon 4.2 supports free text search. Would be really nice to have that in elk

**Comment by Sma11X** [comment]
Do you mean search the posts ? It's supported but not mention by the tips. (difficult to explore for now)

**Comment by audunmb** [comment]
> Do you mean search the posts ? It's supported but not mention by the tips. (difficult to explore for now)

it doesn't work. If you write some text and press enter it doesn't search for the text.

**Comment by haayman-imagem** [comment]
> > Do you mean search the posts ? It's supported but not mention by the tips. (difficult to explore for now)
> 
> it doesn't work. If you write some text and press enter it doesn't search for the text.

it doesn't work with enter. The results will appear in the autocomplete dropdown if you wait long enough

**Comment by audunmb** [comment]
> > > Do you mean search the posts ? It's supported but not mention by the tips. (difficult to explore for now)
> > 
> > 
> > it doesn't work. If you write some text and press enter it doesn't search for the text.
> 
> it doesn't work with enter. The results will appear in the autocomplete dropdown if you wait long enough

you're right, but with possibly hundreds of results I have to scroll very far in a small dropdown to see it. I had to look twice to even see the scrollbar.
